velox/docs/functions/spark/string.rst

```diff
@@ -8,6 +8,11 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
     Returns unicode code point of the first character of ``string``. Returns 0 if ``string`` is empty.
 
+.. spark:function:: bit_length(string) -> integer
+
+    Returns the bit length for the specified string column.::
```

Comment:
Can we mention binary as well? And it's better to add below Spark implementation to the PR description.
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala#L2139-L2141

Reply:
Updated, thanks!